db/seeds.rb

```diff
@@ -14,45 +22,59 @@
     title: 'Throne of Eldraine Booster Box',
     description: 'New: A brand-new, unused, unopened, undamaged item (including handmade items).',
     starting_price: 8500,
-    ends_at: '2019-11-01 15:35:21 -0500'
+    ends_at: '2019-11-01 15:35:21 -0500',
+    user_id: 1
```

Comment:
You shouldn't hardcode the ids since you can't really guarantee that's what you're going to get.

A better approach would be to first create the user objects, then use the real objects with the real ids to create the auctions.

```
users = users_data.each {|user| User.find_or_create_by(user_data)}

auctions_data = [
  {
     title: 'Throne of Eldraine Booster Box',
    ....
     user_id: users[0].id
```

Reply:
Yeah, I meant to point this out. Good idea - I just had to make sure I used `.map` instead of `.each` for this.